bpfman/src/config.rs

```diff
@@ -56,20 +69,20 @@ impl FromStr for Config {
 }
 
 #[derive(Debug, Deserialize, Copy, Clone)]
-pub struct InterfaceConfig {
-    pub xdp_mode: XdpMode,
+pub(crate) struct InterfaceConfig {
+    pub(crate) xdp_mode: XdpMode,
```

Comment:
Should this be exposed through an access function like `interfaces`, `signing` and `database` above?

Reply:
For continuity yes I'll do this